### PR TITLE
Added PnP.PowerShell Set-PnPSite cmdlet

### PR DESCRIPTION
**Pull request by Markus-Hanisch**
Added PnP.PowerShell Set-PnPSite cmdlet to allow custom scripts on a specific SharePoint site:
```powershell
Set-PnPSite <SiteURL> -NoScriptSite $false
```
Further info: [Set-PnPSite](https://docs.microsoft.com/en-us/powershell/module/sharepoint-pnp/set-pnpsite?view=sharepoint-ps)

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/allow-or-prevent-custom-script.md [hunk 1]
```diff
@@ -89,7 +89,12 @@ To allow custom script on a particular site (previously called "site collection"
 3. Run the following command.
 
     ```PowerShell
-    Set-SPOsite <SiteURL> -DenyAddAndCustomizePages 0
+    Set-SPOSite <SiteURL> -DenyAddAndCustomizePages 0
+    ```
+    or by means of PnP.PowerShell cmdlet [Set-PnPSite](https://docs.microsoft.com/en-us/powershell/module/sharepoint-pnp/set-pnpsite?view=sharepoint-ps)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/allow-or-prevent-custom-script.md:94]:
```suggestion
    or by means of the PnP.PowerShell cmdlet [Set-PnPSite](https://docs.microsoft.com/en-us/powershell/module/sharepoint-pnp/set-pnpsite?view=sharepoint-ps)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @Markus-Hanisch, please commit my suggested change.

**Comment by Markus-Hanisch** [comment]
@JohanFreelancer9 done

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**